ui/ui-components/components/setupSteps/setupStepCard.tsx

```diff
@@ -11,15 +14,18 @@ export default function SetupStepCard({
   execApi: Function;
   reload: Function;
 }) {
+  const [loading, setLoading] = useState(false);
   const [activeKey, setActiveKey] = useState(
     step.skipped || step.complete ? null : "0"
   );
 
   async function skip() {
+    setLoading(true);
     await execApi("put", `/setupStep/${step.id}`, {
       skipped: !step.skipped,
     });
-    reload();
+    await reload();
+    setLoading(false);
```

Comment:
If either of these awaits fail, it will never stop loading. Maybe throw into a `try {} finally {}`?